readme.md

```diff
@@ -0,0 +1,278 @@
+# <div align="center"> Express Slow Down </div>
+
+<div align="center">
+
+[![tests](https://github.com/express-rate-limit/express-slow-down/actions/workflows/ci.yaml/badge.svg)](https://github.com/express-rate-limit/express-slow-down/actions/workflows/ci.yaml)
+[![npm version](https://img.shields.io/npm/v/express-slow-down.svg)](https://npmjs.org/package/express-slow-down 'View this project on NPM')
+[![npm downloads](https://img.shields.io/npm/dm/express-slow-down)](https://www.npmjs.com/package/express-slow-down)
+
+Basic rate-limiting middleware for Express that slows down responses rather than
+blocking them outright. Use to slow repeated requests to public APIs and/or
+endpoints such as password reset.
+
+Plays nice with (and built on top of)
+[Express Rate Limit](https://npmjs.org/package/express-rate-limit)
+
+</div>
+
+### Stores
+
+The default memory store does not share state with any other processes or
+servers. It's sufficient for basic abuse prevention, but an external store will
+provide more consistency.
+
+express-slow-down uses
+[express-rate-limit's stores](https://github.com/express-rate-limit/express-rate-limit#store)
```

Comment:
Op, we'll want to update this after https://github.com/express-rate-limit/express-rate-limit/pull/411 goes live